packages/css/src/index.ts

```diff
@@ -238,12 +249,12 @@ const responsive = (styles: Exclude<ThemeUIStyleObject, UseThemeFunction>) => (
     for (let i = 0; i < value.slice(0, mediaQueries.length).length; i++) {
       const media = mediaQueries[i]
       if (!media) {
-        next[key] = value[i]
+        next[key] = value[i] || undefined
```

Comment:
Yeah, so `value[i]` can be `null` here, and we don't allow `null` values in `ThemeUICSSObject` :/
We could, I guess... since Emotion does.